babel.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   presets: [
     '@babel/preset-env',
-    '@babel/react',
+    '@babel/preset-react',
```

Comment:
just standarization here. I prefer this way so when you look at package.json file and try to find where in the project a dependecy is used it appear in search result